website/docs/components/form/super-select/partials/code/how-to-use.md

```diff
@@ -153,18 +148,20 @@ To group similar sets of options, pass a nested data structure specifying the `g
 </Hds::Form::SuperSelect::Single::Field>
 ```
 
-##### Rich content in SuperSelect::Multiple
+#### Multi-select
```

Comment:
The two component types are "Super Select Single" and "Super Select Multiple". We should avoid using a different hyphenated term to refer to them or it could cause confusion.